src/renderer/helpers/assetHelper.ts

```diff
@@ -91,6 +91,12 @@ export const isBtcAsset = (asset: Asset): boolean => eqAsset.equals(asset, Asset
  */
 export const isEthAsset = (asset: Asset): boolean => eqAsset.equals(asset, AssetETH)
 
+/**
+ * Check whether an asset is an ERC20 asset
+ */
+export const isERC20Asset = (asset: Asset): boolean =>
+  ERC20Assets.filter((erc20Asset) => eqAsset.equals(asset, erc20Asset)).length > 0
```

Comment:
We can't use hardcoded `ERC20Assets`, as you know it's just for testnet. We should do it similar to `xchain-eth`...